server/config/passport.js

```diff
@@ -27,4 +32,17 @@ passport.use(new LocalStrategy({
       })
       .catch(err => cb(err));
   }
-));
\ No newline at end of file
+));
+
+passport.use(new JWTStrategy({
+    jwtFromRequest: function(req) {
+        let token = null;
+        if (req && req.cookies) {
+            token = req.cookies['auth'];
+        }
+        return token;
+    },
+    secretOrKey: 'your_jwt_secret'
+}, function(jwt_payload, done) {
+    done(null, jwt_payload);  // Giving that there was no error and the payload itself to access user data
+}));
```

Comment:
This follows documentation from http://www.passportjs.org/packages/passport-jwt/